iCalRetriever.py

```diff
@@ -0,0 +1,59 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.touch_actions import TouchActions
+from time import sleep
+
+driver = webdriver.Chrome()
+# driver.implicitly_wait(1)
+# Loading the hyper planning site
+driver.get("https://hplanning2016.umons.ac.be/invite?fd=1")
+
+try:
+    f = open("links", "w")
+
+    # We wait to be sure that the entire page is loaded.
+    # Without that, Selenium does not find the IDs.
+    try:
+        for i in range(0, 245):
+            print(str(i))
+            # The drop down menu with the list of sections
+            dropDown = WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.ID, "GInterface.Instances[1].Instances[1].bouton")))
+            # Once the drop down menu is found, we click on it.
+            dropDown.click()
+
+            # We must wait that the page loads the sections
+            if i >= 10:
```

Comment:
Parce que si on clique à la première itération, le 1er bouton de la liste n'est plus cliquable.
J'ai pris 10 par sécurité (mais je peux réduire à 2, si tu veux)